### DISCUSSION TITLE
custom filter column with select multiple with checkboxes and above textbox to filter list items just like combo box filter

### DISCUSSION THREAD

**Discussion by SRGajjala** [KevinVandy/material-react-table#15] [Ideas]
How to make a custom filter column with select multiple with checkboxes and above textbox to filter list items just like combo box filter

**Comment by KevinVandy** [comment]
you can code your own filter component and set it on the `Filter` property of a column. 

This is my only example so far, but there should be more docs around this soon: https://www.material-react-table.dev/?path=/story/features-filtering-examples--custom-filter-component
